src/tests/hook.test.ts

```diff
@@ -1,5 +1,5 @@
 import {
-    _createHookController, _getHookController, _getPanelHooksController, _addSetToHookController
+    _createHookController, _getHookController, _getPanelHooksController, _addSetToHookController, _getStatusController
```

Comment:
```suggestion
    _createHookController, _getHookController, _getPanelHooksController, _addSetToHookController
```

Reply:
Pull most recent, I think I pushed right as you started reviewing oops